order.go

```diff
@@ -95,6 +95,18 @@ type Order struct {
 	Refunds               []Refund         `json:"refunds"`
 	UserId                int              `json:"user_id"`
 	OrderStatusUrl        string           `json:"order_status_url"`
+	Gateway               string           `json:"gateway"`
```

Comment:
I'm not sure we should add deprecated fields. I've [mentioned this earlier](https://github.com/getconversio/go-shopify/pull/30#issuecomment-326590817), but I guess it could be up to the user of the library to decide whether to use it or not.